Code/Database/SQLCreateQuery.sql

```diff
@@ -89,14 +89,16 @@ CREATE TABLE Copy (
 CREATE TABLE Borrow (
```

Comment:
Loan

Reply:
this ill fix (idk if in this pr is good though)